<launch>

  <arg name="marker_size" default="9" />
  <arg name="max_new_marker_error" default="0.08" />
  <arg name="max_track_error" default="0.2" />
  <arg name="max_frequency" default="15" />

  <node name="ar_track_alvar_front" pkg="ar_track_alvar" type="individualMarkersNoKinect" respawn="false" output="screen">
    <param name="marker_size"           type="double" value="$(arg marker_size)" />
    <param name="max_new_marker_error"  type="double" value="$(arg max_new_marker_error)" />
    <param name="max_track_error"       type="double" value="$(arg max_track_error)" />
    <param name="max_frequency" 				type="double" value="$(arg max_frequency)" />
    <param name="output_frame"          type="string" value="front_camera_link" />
    <remap from="ar_pose_marker"        to="/front_camera/ar_pose_marker" />
    <remap from="camera_image"          to="/front_camera/color/image_raw" />
    <remap from="camera_info"           to="/front_camera/color/camera_info" />
  </node>

  <node name="ar_track_alvar_rear" pkg="ar_track_alvar" type="individualMarkersNoKinect" respawn="false" output="screen">
    <param name="marker_size"           type="double" value="$(arg marker_size)" />
    <param name="max_new_marker_error"  type="double" value="$(arg max_new_marker_error)" />
    <param name="max_track_error"       type="double" value="$(arg max_track_error)" />
    <param name="max_frequency" 				type="double" value="$(arg max_frequency)" />
    <param name="output_frame"          type="string" value="rear_camera_link" />
    <remap from="ar_pose_marker"        to="/rear_camera/ar_pose_marker" />
    <remap from="camera_image"          to="/rear_camera/color/image_raw" />
    <remap from="camera_info"           to="/rear_camera/color/camera_info" />
  </node>

</launch>
